src/java_tools/junitrunner/java/com/google/testing/coverage/BUILD

```diff
@@ -77,7 +77,7 @@ genrule(
     ],
     outs = ["JacocoCoverage_jarjar_deploy.jar"],
     cmd = "\n".join([
-        "JARJAR=\"$$(mktemp)\"",
+        "JARJAR=\"$$(mktemp -t bazel.XXXXXXXX)\"",
```

Comment:
the way we do it usually is `mktemp -t ${TMPDIR:-/tmp}/bazel.XXXXXXXX` IIRC there was some system unhappy if the template does not include the full directory